### PR TITLE
DFPL-552 Change of wording on upload order function for LA & removal of supporting doc for CMO

### PR DESCRIPTION
**Pull request by chak-shing-lo-justice**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-552

### Change description ###

- Rename the draft order C21 as ADDITIONAL ORDERS (PDO ETC)
- CMO Upload flow- remove case summary and supporting documents - this will be delivered in the pre-requisite ticket above.
- This covers all user roles in FPL and the states from submitted onward.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json [hunk 1]
```diff
@@ -1354,21 +1354,6 @@
...
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "cmoSupportingDocs",
-    "UserRole": "caseworker-publiclaw-solicitor",
-    "CRUD": "CRUD",
-    "Comment": "Requires idam role otherwise cannot add to"
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "cmosSentToJudge",
```
Comment (jamesrferguson1) [inline ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json:1368]: why do we want to remove this?

Comment (jamesrferguson1) [inline ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json:1368]: why do we want to remove this?

File: service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/HearingOrder.java [hunk 2]
```diff
@@ -54,6 +54,10 @@ public class HearingOrder implements RemovableOrder, AmendableOrder, Translatabl
     private CMOStatus status;
     private String judgeTitleAndName;
     private String requestedChanges;
+    // DFPL-552 the uploading option is removed, but user can still access exiting Case Summary
```
Comment (jamesrferguson1) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/HearingOrder.java:57]: Remember to take this comment out :)

Comment (jamesrferguson1) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/HearingOrder.java:57]: Remember to take this comment out :)

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderService.java [hunk 3]
```diff
@@ -158,25 +148,23 @@ public UUID updateCase(UploadDraftOrdersData eventData, List<Element<HearingBook
...
-            List<Element<SupportingEvidenceBundle>> supportingDocs = eventData.getCmoSupportingDocs();
-
             Element<HearingOrder> order = element(from(
                 cmo,
                 hearing.getValue(),
                 time.now().toLocalDate(),
                 eventData.isCmoAgreed() ? AGREED_CMO : DRAFT_CMO,
-                supportingDocs,
+                // DFPL-552 the uploading option is removed, but user can still access exiting Case Summary
+                // DFPL-526 Future case summaries are collected and put in the new 'Hearing Document' tab .
```
Comment (jamesrferguson1) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderService.java:157]: and here :)

Comment (jamesrferguson1) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderService.java:157]: and here :)

File: ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json (hunk 1 above)
Comment (chak-shing-lo-justice) [reply ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json]: Bullseye! That's just a left over, we don't need to remove this. I have reverted this change.

File: service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/HearingOrder.java (hunk 2 above)
Comment (chak-shing-lo-justice) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/HearingOrder.java]: Ok, i have taken out the  comment

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderService.java (hunk 3 above)
Comment (chak-shing-lo-justice) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderService.java]: Ok, i have taken out the  comment

**Review by so99y — APPROVED** [review]

**Pull request merged**